### PR TITLE
feat(replays): Update Replay List onboarding landing page

### LINKED ISSUE #41411: Lazy Load onboarding hero images
Across the app we have these onboarding pages with really great and detailed svg images:

Image

the issue is that they're kind of large, and you don't see them all the time. Probably not even regularly. 

We should lazy load these images to keep them out of the bundle for most users. But loading them into the page shouldn't slow things down for users who are new and are setting things up for the first time.

### PR DESCRIPTION
**Pull request by ryan953**
Added a hero image to the Replay Onboarding experience

It looks fine in most cases. Here are some examples:

1. Here's how it looks at the smallest breakpoint (nav is across top of screen):
  Screen Shot 2022-11-15 at 12 09 53 PM

2. The image is hard to size properly near breakpoints:
  Screen Shot 2022-11-15 at 12 09 48 PM

Fixes #41411

### CODE REVIEW

File: static/app/views/replays/list/replayOnboardingPanel.tsx
Comment (ryan953) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:3]: i'm still waiting on the svg version of the replay-specific image.

File: static/app/views/replays/list/replayOnboardingPanel.tsx
Comment (ryan953) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:51]: all this css might change once we get the replay specific svg in there.

**Comment by ryan953** [comment]
cc @Jesse-Box if you want to checkout this branch and tweak some css near the breakpoints that could help.

It looks fine in most cases, but the image bleeds out of the box just before the breakpoints step down to a smaller size. (see example in description). I think it's acceptable, but not perfect. #pixelsmatter

File: static/app/views/replays/list/replayOnboardingPanel.tsx
Comment (billyvg) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:3]: How big is this image? Can we lazy load it?

**Review by billyvg — COMMENTED** [review]
Codewise this looks good to me.

TBH the image overflowing looks intentional (especially with the traffic cones tipping over) and I'd prefer it over having it inside the box lol but I'll leave final stamp up to @Jesse-Box

File: static/app/views/replays/list/replayOnboardingPanel.tsx
Comment (ryan953) [reply static/app/views/replays/list/replayOnboardingPanel.tsx:3]: We should lazy load these, but none of the other onboarding screens are doing that. I'll follow up with a PR for all of them.

Comment (ryan953) [reply static/app/views/replays/list/replayOnboardingPanel.tsx:3]: Confirmed that the imports resolve to be public urls, the images are not part of the bundle. So there's nothing to do here.

**Review by billyvg — APPROVED** [review]
Should we wait for design approval? I'm good either way.

**Comment by ryan953** [comment]
i'll wait a beat and send it out tmrw at the latest

**Pull request merged**
